enterprise/app/views/api/v1/accounts/applied_slas/download_conversations.csv.erb

```diff
@@ -0,0 +1,26 @@
+<% headers = [
+  I18n.t('reports.sla_csv.conversation_id'),
+  I18n.t('reports.sla_csv.sla_policy_breached'),
+  I18n.t('reports.sla_csv.assignee'),
+  I18n.t('reports.sla_csv.team'),
+  I18n.t('reports.sla_csv.inbox'),
+  I18n.t('reports.sla_csv.labels'),
+  I18n.t('reports.sla_csv.conversation_link'),
+  I18n.t('reports.sla_csv.breached_events')
+] %>
+<%= CSV.generate_line headers %>
+
+<% @breached_slas.each do |sla| %>
+  <% breached_events = sla.sla_events.map(&:event_type).join(', ') %>
+  <% conversation = sla.conversation %>
+  <%= CSV.generate_line([
+    conversation.display_id,
+    sla.sla_policy.name,
+    conversation.assignee.name,
+    conversation.team&.name,
```

Comment:
Please add default value `nil` in case values are not present.

Reply:
The `team&.name` should handle it, in case team is `nil`, the statement will return `nil`

Reply:
I think we should handle this.
![CleanShot 2024-03-25 at 19 21 04@2x](https://github.com/chatwoot/chatwoot/assets/12408980/8915e8ab-200f-4005-985c-86e6bd49a19c)